SmartDeviceLink/SDLLifecycleConfiguration.h

```diff
@@ -123,6 +124,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (copy, nonatomic, nullable) NSArray<Class<SDLSecurityType>> *securityManagers;
 
+/**
+ *  What encryption level video/audio streaming should be. The default is SDLStreamingEncryptionFlagAuthenticateAndEncrypt.
+ */
+@property (assign, nonatomic) SDLStreamingEncryptionFlag streamingEncryption;
+
+/**
+ *  Properties to use for applications that utilitze the video encoder for streaming.
+ */
+@property (copy, nonatomic, nullable) NSDictionary<NSString *, id> *videoEncoderSettings;
```

Comment:
The goal for SMM is to not be interacted with directly by the developer for initialization, only for observation. We want to set these settings in the lifecycle configuration so that the LCM can handle the initialization.